web/src/components/tasks/business-formation/review/ReviewWillPracticeLaw.tsx

```diff
@@ -1,25 +0,0 @@
-import { ReviewNotEntered } from "@/components/tasks/business-formation/review/section/ReviewNotEntered";
-import { ReviewSubSection } from "@/components/tasks/business-formation/review/section/ReviewSubSection";
-import { useConfig } from "@/lib/data-hooks/useConfig";
-import { ReactElement } from "react";
-
-interface Props {
-  willPracticeLaw: boolean | undefined;
-}
-
-export const ReviewWillPracticeLaw = (props: Props): ReactElement => {
-  const { Config } = useConfig();
-
-  const displayResponse = (): ReactElement => {
-    if (props.willPracticeLaw === undefined) {
-      return <ReviewNotEntered />;
```

Comment:
Pebble - why did we remove the logic to render? `ReviewNotEntered`

Reply:
ReviewLineItem has logic inside it that renders a ReivewNotEntered when applicable.